src/options/options.html

```diff
@@ -1,70 +1,128 @@
+                Points.
+              </p>
+              <div
+                class="pb-6 aspect-h-4 aspect-w-3 hidden overflow-hidden rounded-lg lg:block"
+              >
+                <!-- <img src="../images/screenshot.png" alt="Two each of gray, white, and black shirts laying flat." class="h-full w-full object-cover object-center"> -->
+              </div>
+              <p x-show="!showLogin" x-cloak class="text-base text-gray-900">
+                You are signed in to Orbit.
+                If you wish you sign out, visit your
+                <a href="https://app.orbit.love/user/settings/authorized_applications" rel="noreferrer" target="_blank">User Settings on Orbit</a> & revoke access for the "Orbit Browser Extension" application.
+              </p>
+              <p x-show="showLogin" x-cloak class="text-base text-gray-900">
+                To proceed with the installation, please sign in to Orbit.
+              </p>
+            </div>
+            <div x-show="showLogin" x-cloak class="py-4 flex justify-end">
+              <button
+                type="button"
+                class="inline-flex items-center gap-x-2 rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+                @click="startOAuthFlow"
+              >
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  viewBox="0 0 20 20"
+                  fill="currentColor"
+                  class="w-5 h-5"
+                >
+                  <path
+                    fill-rule="evenodd"
+                    d="M2 10a.75.75 0 01.75-.75h12.59l-2.1-1.95a.75.75 0 111.02-1.1l3.5 3.25a.75.75 0 010 1.1l-3.5 3.25a.75.75 0 11-1.02-1.1l2.1-1.95H2.75A.75.75 0 012 10z"
+                    clip-rule="evenodd"
+                  />
+                </svg>
 
-        <p x-show="authenticationCheckStatus.status !== undefined" x-text="authenticationCheckStatus.message"
-          x-bind:class="authenticationCheckStatus.status == 'success' ? 'text-green-500' : authenticationCheckStatus.status == 'warning' ? 'text-yellow-600' : 'text-red-500'">
-        </p>
+                Next: Sign in to Orbit
+              </button>
+            </div>
+          </section>
+          <section
+            x-cloak
+            x-show="['success', 'warning'].includes(authenticationCheckStatus.status) && workspaces && workspaces.length> 0"
+          >
+            <div class="py-5">
+              <label class="text-base font-semibold text-gray-900"
+                >Select a workspace</label
+              >
+              <p class="text-sm text-gray-500">
+                This workspace will be used to retrieve information about community members, and create new ones.
+              </p>
+              <fieldset class="mt-4">
```

Comment:
This is coming in from Tailwind UI: https://tailwindui.com/components/application-ui/forms/radio-groups